src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/SwaggerUI.cs

```diff
@@ -73,10 +77,16 @@ public ISwaggerUI AddServer(HttpRequest req, string routePrefix, IOpenApiConfigu
         }
 
         /// <inheritdoc />
-        public async Task<ISwaggerUI> BuildAsync()
+        public async Task<ISwaggerUI> BuildAsync(IUiCustomizationProvider uiCustomizationProvider = null)
         {
             var assembly = Assembly.GetExecutingAssembly();
 
+            if (uiCustomizationProvider != null)
+            {
+                this._swaggerUiCustomCss = await uiCustomizationProvider.GetStylesheetAsync();
```

Comment:
**question:** Should we try/catch around each of these and instead write a comment in the HTML in the respective language with the exception message so external code can never cause the UI load to fail?